test/parser.spec.js

```diff
@@ -41,9 +41,9 @@ describe('Parser', function() {
       '\n# tar' +
       '\n> archiving utility' +
       '\n' +
-      '\n- create an archive' +
+      '\ncreate an archive' +
```

Comment:
I think that's the work of the linter. The client is not bothered about it. It will display verbatim.